### ISSUE TITLE
Reference to cloudscribe.Pagination.Models

### ISSUE THREAD

**Issue by micheltol** [cloudscribe/cloudscribe.Web.Pagination#41]
Hi,

is there a reason to have a reference from cloudscribe.Web.Pagination to the cloudscribe.Pagination.Models project. As far as I can see in the code and the documentation this reference is not needed.

tnx!

**Comment by joeaudette** [comment]
Giving it some thought, maybe it is time to bump the major version of the taghelper lib to 3.0 and remove that ref, and bump the models to 2.0 and remove the unwanted classes.

**Comment by joeaudette** [comment]
thing is PagedResult of T is also used in web view models, so it is convenient for it to be pulled in automatically along with the taghelper, and removing it will break existing apps that update to the new nuget if they use it in viewmodels and that will likely add support burden for me and inconvenience to consumers all for one little model class. 

I think I will keep that ref but remove the extra classes that have been marked obsolete for a long time.

**Comment by micheltol** [comment]
tnx for the feedback; I would suggest to remove the reference. 
Keeping a reference-footprint as small as possible is always a good thing.
The pager tag helper library is not depending on it; so why keep it?

> Giving it some thought, maybe it is time to bump the major version of the taghelper lib to 3.0 and remove that ref, and bump the models to 2.0 and remove the unwanted classes.

I don't have a overview of all the projects / libs you are supporting, but my feeling says it's bunch o them.  I would bump the versions and remove the reference. It would be semver proof. If people ar eupgrading their package they can expect breaking changes when upgrading a major version.

**Comment by joeaudette** [comment]
I have removed the unwanted classes from the models project so now there is just the single model class in that library which I do not think is an unwarranted burden to pull along with the pager tag helper library.

My decision is to keep the ref so as to avoid breaking web apps that use the PagedResult of T in viewmodels but that don't already have a direct ref to the models project. I have only bumped the patch versions and I have published updated nugets.
